src/sync_handlers/bonds_handler.ts

```diff
@@ -0,0 +1,18 @@
+import {BondSyncHandler} from "./bonds_sync_handler";
+import {NewBondsInfo} from "../models/bonds";
+
+export class BondHandler {
+  private bondSyncHandler = new BondSyncHandler();
+
+  routeNewBondsInfo(bondsInfo: NewBondsInfo) {
```

Comment:
Possible improvement would be to first check whether bond has multiple prices, in which case we skip the current bond, otherwise we get and add bond info.

Reply:
We'd still have to access the specific bond info for which we want to check the number of prices, so this is not a significant improvement.